scripts/close-issue-channel/README.md

```diff
@@ -0,0 +1,22 @@
+# Close Issue Channel Script
+
+## Command Line Usage
+
+1. Make sure your environment variables are defined.
+1. Run the following command.
+
+```bash
+node scripts/close-issue-channel --payload="{ \"number\": 180 }"

```

Comment:
Why not just do `--issue 180` and pass in `--issue ${{ github.event.issue.number }}`?

Reply:
Nice catch. The issue number is all that's needed for now.